### PR TITLE
All Python codes for last week POTD's!

### PR DESCRIPTION
**Pull request by thepratholic**

### CODE REVIEW

File: Adding Spaces to a String/Python.py
Comment (AditiChourasia) [inline Adding Spaces to a String/Python.py:1]: File names should contain question number too. Just copy the problem name from leetcode itself and paste it.
It should be in the format "XXX. question name".
This is done to make sure when we add other language files, they go in the same question folder.

Follow it for other questions as well.

**Comment by thepratholic** [comment]
Okay Di!
I will be careful from tomorrow. Thanks for correcting me✌🏻.

**Comment by AditiChourasia** [comment]
Hii Pratham,

Please correct the folder names in the current Pull request too.

Thanks,
Aditi

**Comment by thepratholic** [comment]
I have updated it.

**Pull request merged**